x-pack/plugins/security_solution/cypress/e2e/detection_rules/rules_table_auto_refresh.cy.ts

```diff
@@ -43,6 +43,10 @@ describe.skip('Alerts detection rules table auto-refresh', () => {
     }
   });
 
+  beforeEach(() => {
+    login();
```

Comment:
Same here: what's the point of the login in the beforeEach clause if we are logging in in the before clause above?
I guess you need to be logged in to create the rules, but if we are already logged in, why log in again before each test? Is that causing flakyness in any way?